version.bzl

```diff
@@ -1,3 +1,3 @@
 # NOTE: If you change the format of this line, you must change the bash command
 # in /scripts/publish to extract the version string correctly.
-DF_VERSION = "2.1.0"
```

Comment:
hm. should we now be making a `main_v2` to be able to push bugfixes (security fixes) to v2 users?

Reply:
Fair point, more effort than it's worth currently - will hold off submitting this